src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -547,9 +556,16 @@ function featuresToArray(config: DevContainerConfig, additionalFeatures: Record<
 
 // Process features contained in devcontainer.json
 // Creates one feature set per feature to aid in support of the previous structure.
-async function processUserFeatures(output: Log, config: DevContainerConfig, workspaceRoot: string, env: NodeJS.ProcessEnv, userFeatures: DevContainerFeature[], featuresConfig: FeaturesConfig, skipFeatureAutoMapping: boolean): Promise<FeaturesConfig> {
+async function processUserFeatures(params: ContainerFeatureInternalParams, config: DevContainerConfig, workspaceRoot: string, userFeatures: DevContainerFeature[], featuresConfig: FeaturesConfig): Promise<FeaturesConfig> {
+	const { platform, output, skipFeatureAutoMapping, env, } = params;
+
+	let configPath = config.configFilePath.path;
```

Comment:
`config.configFilePath.fsPath` should give you the local path.

Reply:
In codespaces, it gives a `vscode-remote://` URI - is that what we want?